src/components/TrackPain/NewPainEntry.js

```diff
@@ -534,12 +536,20 @@ class NewPainEntry extends Component {
               </FormControl>
             </div>
             <div>
+              <FormLabel
+                component="legend"
+                className={classes.notesLegendStyle}
+              >
+                Include here any notes related to the pain, for example, can you
+                identify any event that could triggered it? what did you do
+                different than other days?
```

Comment:
Include here any notes related to the pain. For example, can you identify any event that could trigger it? What did you do different than other days?

Reply:
Done